aria/orchestrator/workflows/api/task.py

```diff
@@ -65,86 +65,82 @@ def __init__(self,
                  actor,
                  actor_type,
                  interface_name,
-                 operation_name,
-                 runs_on=None,
+                 operation_name, # remove configuration
+                 inputs=None,
                  max_attempts=None,
                  retry_interval=None,
-                 ignore_failure=None,
-                 inputs=None):
+                 ignore_failure=None):
         """
         Do not call this constructor directly. Instead, use :meth:`for_node` or
         :meth:`for_relationship`.
         """
 
-        assert isinstance(actor, (models.Node, models.Relationship))
-        assert actor_type in ('node', 'relationship')
         assert interface_name and operation_name
-        assert runs_on in models.Task.RUNS_ON
         super(OperationTask, self).__init__()
 
         self.actor = actor
+        self.actor_type = actor_type
```

Comment:
Because we decided together that the logic that was here should not be in operation creation, but in service instance creation.